delta/service/src/test/resources/serialization/schema-updated.json

```diff
@@ -80,7 +80,7 @@
     ]
   },
   "expanded": [
-    {
+    [{
```

Comment:
is the list within the list expected?

Reply:
Yes. This is just the internal representation of the event. is a `NonEmptyList[ExpandedJsonLd]`